### DISCUSSION TITLE
windows下启动编辑器报错

### DISCUSSION THREAD

**Discussion by wanzaixiaoxin** [ejoy/ant#144] [Q&A]
024-05-11 16:08:58.60][INFO ]( ant.window|window )(E:/ant/pkg/ant.render/viewport/state.lua:49) device width/hegiht: 1.93
[2024-05-11 16:08:58.64][ERROR]( ant.window|window ) assertion failed!
stack traceback:
        ( service:6 )
        E:/ant/pkg/ant.render/viewport/state.lua:27: in upvalue 'calc_scene_size'
        E:/ant/pkg/ant.render/viewport/state.lua:54: in upvalue 'resize'
        E:/ant/pkg/ant.render/viewport/state.lua:67: in field 'set_device_viewrect'
        E:/ant/tools/editor/pkg/tools.editor/widget/main_view.lua:124: in upvalue 'handle_main_view'
        E:/ant/tools/editor/pkg/tools.editor/widget/main_view.lua:146: in field 'show'
        E:/ant/tools/editor/pkg/tools.editor/gui_system.lua:79: in local 'func'
        E:/ant/pkg/ant.world/main.lua:378: in upvalue 'update_func'
        E:/ant/pkg/ant.world/main.lua:403: in field '_pipeline_update'
        E:/ant/pkg/ant.world/main.lua:587: in method 'pipeline_update'
        E:/ant/pkg/ant.window/service/window.lua:79: in upvalue 'f'

**Comment by cloudwu** [comment]
下次建议学习一下 markdown 稍微排版一下，见
https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-code  （如何引用代码)

**Comment by constly** [comment]
我这边更到最新后，启动编辑器倒是没报错，关闭时有上面类似的报错

**Comment by cloudwu** [comment]
https://github.com/ejoy/ant/pull/143

是这个？
